kolibri/plugins/setup_wizard/assets/src/views/progress-toolbar/index.vue

```diff
@@ -2,7 +2,7 @@
 
   <ui-toolbar
     class="progress-toolbar"
-    navIcon="arrow_back"
+    :navIcon="isRtl ? 'arrow_forward' : 'arrow_back'"
```

Comment:
here also, maybe prefer adding the `rtl-icon` class instead of switching icon names?